packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/SelectDemo/SelectDemo.tsx

```diff
@@ -47,7 +50,15 @@ export class SelectDemo extends Component<SelectDemoState> {
     plainTypeaheadMultiIsPlain: true,
     customTypeaheadMultiIsExpanded: false,
     customTypeaheadMultiSelected: [],
-    direction: SelectDirection.down
+    direction: SelectDirection.down,
+    typeaheadOptions: [
+      { value: 'Alabama', disabled: false},
+      { value: 'Florida', disabled: false },
+      { value: 'New Jersey', disabled: false }
+    ],
+    typeaheadCreateNew: undefined,
```

Comment:
Why does this need to be set to undefined?

Reply:
It doesn't, I will remove